packages/lib/src/LgIdProps/index.ts

```diff
@@ -7,5 +7,5 @@ export interface LgIdProps {
   /**
    * LG test id passed to the component wrapper.
    */
-  ['data-lgid']?: string;
+  ['data-lgid']?: `lg-${string}`;
```

Comment:
Only concern with this is passing any variables will need to be declared `as const`, otherwise anything typed simply as `string` will error

Reply:
oh i see, i'll revert then